docs/config_guide.md

```diff
@@ -24,6 +24,9 @@ DATABASE_URL_TEST=...
 
 # Postmark API key for various environments
 # Postmark is the service used to send e-mails
+# NOTE: Use "POSTMARK_API_TEST" as the API key in "test" and "dev" environments, this will prevent sending
```

Comment:
Can you remove "dev" environments? Also is it worth linking to postmarks docs rather than our issue?

Reply:
The bit about the test API key isn't totally obvious in the postmark docs, our issue has a screenshot of the right part of the docs.